icu4c/source/common/uresbund.cpp

```diff
@@ -91,6 +91,172 @@ static UBool chopLocale(char *name) {
     return FALSE;
 }
 
+static UBool hasScript(const char* localeID) {
+    UErrorCode err = U_ZERO_ERROR;
+    int32_t scriptLength = uloc_getScript(localeID, NULL, 0, &err);
+    return scriptLength != 0;
+}
+
+static UBool hasRegion(const char* localeID) {
+    UErrorCode err = U_ZERO_ERROR;
+    int32_t scriptLength = uloc_getCountry(localeID, NULL, 0, &err);
```

Comment:
scriptLength is confusing. Maybe regionLength?

Reply:
Bonehead copy/paste error.  Fixed.